intel_pytorch_extension_py/ops/DNNL_linear_relu.py

```diff
@@ -0,0 +1,55 @@
+import torch
+from torch import nn
+from torch.nn.parameter import Parameter
+from torch.nn import init
+from torch.autograd import Function
+import math
+import _torch_ipex as core
+
+class DNNLFC(Function):
+    @staticmethod
+    def forward(ctx, input, weight, bias):
+        output = core.linear_fuse_relu(input, weight, bias)
+        ctx.save_for_backward(input, weight, bias, output)
+        return output
+
+    @staticmethod
+    def backward(ctx, grad_output):
+        input, weight, bias, output = ctx.saved_tensors
+        grad_output = grad_output.contiguous()
+        if bias == None:
+            output_mask = (input.requires_grad, weight.requires_grad, 0)
+        else:
+            output_mask = (input.requires_grad, weight.requires_grad, bias.requires_grad)
+        grad_output = core.dnnl_relu_use_dst_backward(grad_output, output)
+        grad_input, grad_weight, grad_bias = core.linear_backward(input, grad_output, weight, output_mask)
+        return (grad_input, grad_weight, grad_bias)
+
+class DNNLLRFuse(nn.Module):

```

Comment:
How about `DNNLLinearFuseReLU`? "LR" sounds vague to me.

Reply:
Done